Sources/Core/ARM/HardwareBreakpointManager.cpp

```diff
@@ -40,4 +40,9 @@ ErrorCode HardwareBreakpointManager::isValid(Address const &address,
   DS2LOG(Debug, "Trying to set hardware breakpoint on arm");
   return kErrorUnsupported;
 }
+
+size_t HardwareBreakpointManager::chooseBreakpointSize() {
+  DS2BUG(
```

Comment:
`DS2BUG()` is like an assert that stays even in release builds, so I think we shouldn't do that. We should somehow return an error to the caller.

Reply:
How do you think we should return an error to the caller? DS2BUG is a fatal assert indicative of a bug. Maybe the whole "This is FATAL, cannot recover" is a bit extreme. Maybe a warning?